unic/ucd/name/tests/basic_tests.rs

```diff
@@ -13,12 +13,37 @@ extern crate unic_ucd_name;
 use unic_ucd_name::Name;
 
 #[test]
-fn basic_tests() {
+fn test_name_str() {
     assert_eq!(
         Name::of('A').expect("No name for A").to_string(),
```

Comment:
nit: In tests, it's better to not use `expect()`, unless it's something that may fail. I think using `unwrap()` in these cases make the test more readable by being shorter, and more clean about expectation.

Reply:
Sounds good, fixed :)